### PR TITLE
Create biosocieties.csl

### PR DESCRIPTION
**Pull request by damnation333**
see under issues #1721  for style-request.

### CODE REVIEW

**Comment by damnation333** [comment]
Can somebody help with the first issue? Didn't get it to validate because of the </else-if> line 172. Not sure why.

**Comment by rmzelle** [comment]
Did you use the validator? See http://validator.citationstyles.org/?url=https%3A%2F%2Fraw.githubusercontent.com%2Fdamnation333%2Fstyles%2Fmaster%2Fbiosocieties.csl&version=1.0.1

File: biosocieties.csl [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+            <text macro="editor"/>
+            <text variable="container-title" font-style="normal" suffix="."/>
+            <text variable="collection-title" suffix="."/>
+            <text variable="event" suffix="."/>
+            <group suffix="." delimiter=" ">
+              <text macro="publisher" prefix=" " suffix=","/>
+              <text variable="issued" suffix=":">
+              <label variable="page" form="short" suffix=" "/>
+              <text variable="page"/>
+            <else-if>
```
Comment (rmzelle) [inline biosocieties.csl]: @damnation333, there are a bunch of errors here. Just by eye, it looks like the groups started on lines 162 and 168 are both missing a closing tag (`</group>`), the `<else-if>` on line 173 should be a closing tag (`</else-if>`), and "issued" is a date variable, so you can't print it with `<text/>` on line 170. There might be more, but by fixing these issues and using the validator you can probably figure it out.

File: biosocieties.csl [hunk 2]
```diff
@@ -157,20 +157,24 @@
...
-            <group suffix="." delimiter=" ">
-              <text macro="publisher" prefix=" " suffix=","/>
-              <text variable="issued" suffix=":">
-              <label variable="page" form="short" suffix=" "/>
-              <text variable="page"/>
-            <else-if>      
+          <group prefix=" " delimiter=", ">
+            <text variable="container-title" form="short" suffix=","/>
+            <text variable="collection-title"/>
+          </group>
```
Comment (rmzelle) [inline biosocieties.csl]: @damnation333, `suffix=","` on line 163 should be redundant if you already have `delimiter=", "` on the group.

File: biosocieties.csl [hunk 3]
```diff
@@ -157,20 +157,24 @@
...
+          <group prefix=" " suffix=":">
+            <date variable="issued">
+              <date-part name="day" suffix=" " range-delimiter="-"/>
+              <date-part name="month"/>
+            </date>
+          </group>
+          <group suffix="." delimiter=" ">
+            <text macro="publisher" prefix=" " suffix=","/>
+            <label variable="page" form="short" suffix=" "/>
```
Comment (rmzelle) [inline biosocieties.csl]: @damnation333, this `suffix=" "` should also be redundant.

File: biosocieties.csl (hunk 2 above)
Comment (damnation333) [inline biosocieties.csl]: I tried, but it isn't right if I delete it.

Comment (rmzelle) [inline biosocieties.csl]: I guess it could make a difference in cases where you have a "container-title" but not a "collection-title".

**Comment by rmzelle** [comment]
Thanks. I changed the `<issnl/>` to `<issn/>` (ISSN-L hasn't really caught on, so we stick to print and electronic ISSNs), and fixed the style to US English.

**Comment by rmzelle** [comment]
Closes #1721.

@damnation333, will you post to the original Zotero thread to let the requester know their style arrived?

**Comment by damnation333** [comment]
@rmzelle will do. Thanks for merging :)

**Pull request merged**